browser/ui/browser_commands.cc

```diff
@@ -103,9 +103,9 @@ namespace brave {
 namespace {
 
 bool CanTakeTabs(const Browser* from, const Browser* to) {
-  return from != to && !from->IsAttemptingToCloseBrowser() &&
-         !from->IsBrowserClosing() && !from->is_delete_scheduled() &&
-         to->profile() == from->profile();
+  return from != to && from->type() == Browser::TYPE_NORMAL &&
```

Comment:
Nice fix! Had to look up the other types here:
https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/browser.h;l=157-190;drc=41374c974d98f8cf67134f9ddb8d96d398154dfe

We likely also had a bug if someone popped devtools out to a separate window or an extension popup is being shown. This should fix it properly 😄

Reply:
Update: yeah I confirmed before this fix, it would pull the devtools into the main window as a tab 😂